test/src/lib.rs

```diff
@@ -418,6 +424,44 @@ mod tests {
         }
     }
 
+    #[test]
+    #[ignore]
+    fn cwe_252() {
+        let mut error_log = Vec::new();
+        let mut tests = linux_test_cases("cwe_252", "CWE252");
+        let num_expected_occurences = 9;
+        let num_expected_occurences_lkm = 1;
+
+        tests.extend(lkm_test_cases("cwe_252", "CWE252"));
+
+        mark_architecture_skipped(&mut tests, "ppc64");
+        mark_architecture_skipped(&mut tests, "ppc64le");
+        mark_skipped(&mut tests, "aarch64", "gcc");
+        mark_skipped(&mut tests, "mips", "gcc");
+        mark_skipped(&mut tests, "mips64", "gcc");
+        mark_skipped(&mut tests, "mips64el", "gcc");
+        mark_skipped(&mut tests, "mipsel", "gcc");
+        mark_skipped(&mut tests, "x86", "gcc");
+
+        for test_case in tests {
+            let num_expected_occurences = if test_case.is_lkm {
+                num_expected_occurences_lkm
+            } else {
+                num_expected_occurences
+            };
+
+            if let Err(error) = test_case.run_test("[CWE252]", num_expected_occurences) {
+                println!("{:?}", error);
```

Comment:
This `println!` should be unnecessary, because printing is done by `print_errors` later on.